pages/[id].jsx

```diff
@@ -1,33 +1,43 @@
 import Head from 'next/head';
 import apiService from '../services/apiService';
 import getFullName from '../util/getFullName';
+import getParticipantOrLeaderStatus from '../util/getParticipantOrLeaderStatus';
 import { getSkills } from '../util/getSkills';
 import { getPastClients } from '../util/getPastClients';
 import CurrentProjects from '../components/CurrentProjects';
 import Layout from '../components/Layout';
 import PersonalBio from '../components/PersonalBio';
+import PersonalOverview from '../components/PersonalOverview';
 import SkillsGrid from '../components/SkillsGrid';
 import TradingCardImage from '../components/TradingCardImage';
 import PastClients from '../components/PastClients';
 import Show from '../components/Show';
 
 export default function DetailPage({ fetchedTeamMember }) {
+  const cohortStatus = getParticipantOrLeaderStatus(fetchedTeamMember);
   const fullName = getFullName(fetchedTeamMember);
   const skills = getSkills(fetchedTeamMember);
   const pastClients = getPastClients(fetchedTeamMember);
+
   return (
     <Layout>
       <Head>
         <title>{`${fullName} | Sparkbox Team Availability`}</title>
         <meta name="description" content={`View details about ${fullName}, including their projects, skills, and interests.`} />
       </Head>
-      <h1>{`Greetings ${fullName}!`}</h1>
+      <div className="obj-personal-overview">
```

Comment:
Separating personal-overview layout into a it's own class :+1: but I think it would be best to move it into the PersonalOverview component. That way we keep the personal-overview styles scoped to the component. That means TradingCardImage will need to either be moved into PersonalOverview or if you are feeling up to a challenge I think in Reed's foundry article you might find an interesting way to pass TradingCardImage to PersonalOverview as a child

Reply:
Agreed, that's much more elegant